Source/CsDebugScript.Engine/Debugger.cs

```diff
@@ -578,6 +578,59 @@ public static MemoryBuffer ReadMemory(Process process, ulong address, uint size)
             }
         }
 
+        #region DebuggeeControl
+        /// <summary>
+        /// When doing live process debugging continues debugee execution of the current process.
+        /// </summary>
+        public static void ContinueExecution()
+        {
+            Context.Debugger.ContinueExecution(Process.Current);
+        }
+
+        /// <summary>
+        /// When doing live process debugging breaks debugee execution of the current process.
+        /// </summary>
+        public static void BreakExecution()
+        {
+            Context.Debugger.BreakExecution(Process.Current);
+        }
+
+        /// <summary>
+        /// When doing live process debugging continues debugee execution of the specified process.
+        /// </summary>
+        /// <param name="process"></param>
```

Comment:
Done.

---
In reply to: [70408514](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70408514) [](ancestors = 70408514)